pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Functions.java

```diff
@@ -756,6 +756,42 @@ void downloadFunction(String destinationFile, String tenant, String namespace, S
     CompletableFuture<Void> downloadFunctionAsync(
             String destinationFile, String tenant, String namespace, String function);
 
+    /**
+     * Download Function Code.
+     *
+     * @param destinationFile
+     *           file where data should be downloaded to
+     * @param tenant
+     *            Tenant name
+     * @param namespace
+     *            Namespace name
+     * @param function
+     *            Function name
+     * @param extraFunction
+     *            Whether to download the extra-function
```

Comment:
this is somehow not very intuitive, we should cite that this works only if we are referring to a Sink or a Source